src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-stopTimeout.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    Timeout, in seconds, to apply when stopping the container after a job has finished. The default of 10s is fine for
+    most cases. However, if you container needs to do some cleanup after the slave has stopped you can increase the
+    timeout. After the timeout the container will be killed.
```

Comment:
Please rephrase the last sentence.
"If the container fails to stop gracefully before the timeout ends, the container will be killed."

I'd also suggest that you use a new line for every sentence rather than wrapping the text manually (it'll look the same to the end user, but it'll means that any subsequent edits to this file will result in smaller & simpler git commits).